modules/memcached/memcached_test.go

```diff
@@ -0,0 +1,21 @@
+package memcached_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/memcached"
+)
+
+func TestRun(t *testing.T) {
+	ctx := context.Background()
+
+	ctr, err := memcached.Run(ctx, "memcached:1.6-alpine")
+	testcontainers.CleanupContainer(t, ctr)
+	require.NoError(t, err)
+
+	// perform assertions
```

Comment:
question: was this left as a placeholder for more work?

Reply:
It's part of the template. Will remove it in a follow, up , also cleaning it up from the modules where it does not apply anymore